micro_rpc/proto/BUILD

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-load("@rules_java//java:defs.bzl", "java_proto_library")
+load("@rules_java//java:defs.bzl", "java_lite_proto_library", "java_proto_library")
```

Comment:
Javalite will work for both Android and non-Android, right?

Reply:
Correct. I think it'd be worth migrating everything (android and non-android) over to java-lite in a separate commit, though, to keep this one focussed.

Reply:
ended up moving some targets over to javalite so they can be shared with android and thereby reduce build tree duplication